README.md

```diff
@@ -0,0 +1,8 @@
+Workflow course assignment
+
+[![Automated E2E Testing](https://github.com/ohjenssen/workflow-ca/actions/workflows/e2e-test.yml/badge.svg?branch=workflow-automated-testing)](https://github.com/ohjenssen/workflow-ca/actions/workflows/e2e-test.yml)
+[![Automated Unit Testing](https://github.com/ohjenssen/workflow-ca/actions/workflows/unit-test.yml/badge.svg?branch=workflow-automated-testing)](https://github.com/ohjenssen/workflow-ca/actions/workflows/unit-test.yml)
+
```

Comment:
I would expect to see the various commands listed here too like `npm i`, `npm run build`, `npm run start`